text/0016-named-tracers.md

```diff
@@ -14,41 +14,49 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 
 The mechanism of "Named Tracers and Meters" proposed here is motivated by following scenarios:
 
-* For a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+**Expensive instrumentation**
 
-* If a library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with those instrumentation libraries later.
+For an operator of OpenTelemetry, there is currently no possibility of influencing the amount of the data produced by reporting libraries. Reporting libraries can easily "spam" backend systems, deliver bogus data, or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+
+**Reporting library identification**
+
+If a reporting library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with the library which reported them later.
+
+**Conflicting instrumentation libraries**
+
+It is the eventual goal of OpenTelemetry that library vendors implement the OpenTelemetry API, obviating the need to auto-instrument their library. An operator should be able to disable built-in instrumentation even if the library does not provide an explicit configuration to disable instrumentation.
 
 This proposal attempts to solve the stated problems by introducing the concept of:
  * _Named Tracers and Meters_ identified via a **name** (e.g. _"io.opentelemetry.contrib.mongodb"_) and a **version** (e.g._"semver:1.0.0"_) which is associated with the Tracer / Meter and the Spans / Metrics it produces.
- * A `TracerFactory` / `MeterFactory` as the only means of creating a Tracer or Meter.
+ * A `TracerRegistry` / `MeterRegistry` as the only means of creating a Tracer or Meter which stores references to all `Tracers` and `Meters` it has created.
```

Comment:
Why do we need to prescribe storing of references? I personally don't think this is needed or even useful.

Reply:
would you have it create a new tracer every time getTracer is called?

Reply:
I think Christian's intent was to not have the spec define such implementation details.

Reply:
I just removed it. We can have a line like that in the spec if we want.